package.json

```diff
@@ -49,6 +49,24 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
+    "configuration": [
+      {
+        "title": "Flatpak",
+        "properties": {
+          "flatpak.integrations": {
+            "type": "object",
+            "$comment": "The key is the extension id for the integration",
+            "default": {
```

Comment:
This could may be empty and return true by default in isEnabled if the key is undefined